arch/generic/compare256_c.c

```
@@ -4,6 +4,10 @@
  */
 
 #include "zbuild.h"
```

Cppcheck (2.19.0)

[information] Include file

(missingInclude)